odh-dashboard/base/deployment.yaml

```
@@ -27,7 +27,7 @@ spec:
       serviceAccount: rhods-dashboard
       containers:
       - name: rhods-dashboard
-        image: rhods-dashboard
+        image: $(image-sha-256-odh-dashboard)
```

I'm thinking about changing the name of this parameter (and others) from `$(image-sha-256-odh-dashboard)` to something like `$(odh-dashboard-image)` because it's true that we are using sha-256 but it could be a tag too on a different scenario. What do you think?

```suggestion
        image: $(odh-dashboard-image)
```